.github/workflows/tests.yml

```diff
@@ -0,0 +1,33 @@
+name: tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.4, 2.6]
```

Comment:
should we run on different rubies? I know these are the ones that we _were_ using but do we wanna keep using them?

I guess one step at a time. so let's leave em.